rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -280,23 +280,31 @@ impl SDKClient {
             .await
             .unwrap_or_default();
         let default = Orderbook::<FIFOOrderId, PhoenixOrder> {
```

Comment:
is default a keyword? github is coloring it differently. if it is a reserved term let's rename

Reply:
I do not think that default is a keyword in rust